packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -83,6 +84,14 @@ class Icon extends StatelessWidget {
   /// ```
   final Color color;
 
+  /// Semantic label for the icon.
+  ///
+  /// This would be read out in accessibility modes (e.g TalkBack/VoiceOver).
+  /// Note that this label does not show on the UI.
```

Comment:
nit: "note that.." is against our doc style. Just say "This label does not show up in the UI".